convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -62,143 +63,121 @@ def extract_packages(self, pkg):
         if pkgs_not_removed:
             message = "The following packages were not removed: %s" % ", ".join(pkgs_not_removed)
             logger.warning(message)
             self.add_message(
                 level="WARNING",
-                id="EXCLUDED_PACKAGES_NOT_REMOVED",
-                title="Excluded packages not removed",
-                description="Excluded packages which could not be removed",
+                id="SPECIAL_PACKAGES_NOT_REMOVED",
+                title="Special packages not removed",
+                description="Special packages which could not be removed",
                 diagnosis=message,
             )
+
         if pkgs_removed:
             message = "The following packages will be removed during the conversion: %s" % ", ".join(pkgs_removed)
             logger.info(message)
             self.add_message(
                 level="INFO",
-                id="EXCLUDED_PACKAGES_REMOVED",
-                title="Excluded packages to be removed",
-                description="We have identified installed packages that match a pre-defined list of packages that are"
-                " to be removed during the conversion",
+                id="SPECIAL_PACKAGES_REMOVED",
+                title="Special packages to be removed",
+                description=(
+                    "We have identified installed packages that match a pre-defined list of packages that are"
+                    " to be removed during the conversion"
+                ),
                 diagnosis=message,
             )
 
+        super(RemoveSpecialPackages, self).run()
 
-class RemoveRepositoryFilesPackages(actions.Action):
-    id = "REMOVE_REPOSITORY_FILES_PACKAGES"
-    dependencies = (
-        "BACKUP_REDHAT_RELEASE",
-        # We use the backed up repos in remove_pkgs_unless_from_redhat()
-        "BACKUP_REPOSITORY",
-        # The installation of sub-man pkgs needs access to the original repofiles to get the sub-man deps from there
-        "PRE_SUBSCRIPTION",
-        "BACKUP_PACKAGE_FILES",
-    )
 
-    def run(self):
-        """
-        Remove those non-RHEL packages that contain YUM/DNF repofiles
-        (/etc/yum.repos.d/*.repo) or affect variables in the repofiles (e.g.
-        $releasever).
-
-        Red Hat cannot automatically remove these non-RHEL packages with other
-        excluded packages. While other excluded packages must be removed before
-        installing subscription-manager to prevent package conflicts, these
-        non-RHEL packages must be present on the system during
-        subscription-manager installation so that the system can access and
-        install subscription-manager dependencies. As a result, these non-RHEL
-        packages must be manually removed after subscription-manager
-        installation.
-        """
-        super(RemoveRepositoryFilesPackages, self).run()
+def _remove_packages_unless_from_redhat(pkgs_list):
```

Comment:
Why don't have this as method of `RemoveSpecialPackages`? I see it's called only from the `run` and anywhere else

Reply:
So, I decided to split this method because it is not using any internal reference of the class. To not pollute the class with functions that does not need any internal reference, I decided to put this outside the class scope. 

It is true that we could move this to the class as it belongs only to there, but still, if a function doesn't need any internal references, meaning that in this sense we think of them as a `static function`, I prefer to move them to be in the module level rather than in the class level.